casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -290,11 +290,12 @@ object Genesis {
     for {
       _       <- skFiles
       printer <- Sync[F].delay { new PrintWriter(genBondsFile) }
-      _ <- Foldable[List].foldM[F, (PublicKey, Long), Unit](bonds.toList, ()) {
-            case (_, (pub, stake)) =>
+      _ <- bonds.toList.traverse {
```

Comment:
I think `traverse_` will be a bit more typesafe

Reply:
Good catch! :)